lib/failure_app.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
I _think_ Rails 5+ prefers these files to be in `/app/lib` for autoloading purposes and/or for convention, can we do that?

Reply:
Done.